### PR TITLE
Wix fragments

### PR DESCRIPTION
**Pull request by neodesys**

### CODE REVIEW

**Comment by dabrain34** [comment]
Hello, do you think you gonna upstream this branch later ? In this case I can drop the custom branch I was workin on during the Hackfest

**Comment by neodesys** [comment]
Yes, I will create a MR on upstream as soon as this PR is reviewed and accepted on our side :+1:

File: cerbero/packages/wix_packager.py [hunk 1]
```diff
@@ -57,17 +57,17 @@ def pack(self, output_dir, devel=False, force=False, keep_temp=False):
         return paths
 
     def create_merge_module(self, output_dir, package_type, force, version,
-                            keep_temp):
+                            keep_temp, keep_strip_temp_dir=False):
```
Comment (ylatuya) [inline cerbero/packages/wix_packager.py]: I don't think you need this variable, you only have to keep it when using fragments

File: cerbero/packages/wix.py [hunk 2]
```diff
@@ -222,28 +301,28 @@ def _add_root_dir(self):
 
     def _add_vs_templates(self):
         etree.SubElement(self.module, 'PropertyRef',
-            Id='VS_PROJECTTEMPLATES_DIR')
+                         Id='VS_PROJECTTEMPLATES_DIR')
```
[the remark this answers is not in the document]
Comment (neodesys) [reply cerbero/packages/wix.py]: They were not, it's pep8 formatting automatically when executed on save.
If it's an issue, I can restablish previous indentation ?

File: cerbero/packages/wix_packager.py (hunk 1 above)
Comment (neodesys) [reply cerbero/packages/wix_packager.py]: Yes, but as the method returns different kind of values depending on this variable value (a single string or a duple of strings), I prefer keeping it explicit on method parameters than depending on an internal state.

So you know exactly what kind of data to expect as a return value at the moment of calling the method.

**Pull request merged**